src/core/client/stream/tabs/Comments/Stream/FeaturedComments/FeaturedCommentsContainer.tsx

```diff
@@ -114,6 +117,14 @@ export const FeaturedCommentsContainer: FunctionComponent<Props> = (props) => {
     // If we aren't warned.
     !warned;
 
+  const showGoToDiscussions = useMemo(
+    () =>
+      !!props.viewer &&
+      !!props.settings &&
+      props.settings.featureFlags.includes(GQLFEATURE_FLAG.DISCUSSIONS),
+    [props.viewer, props.settings]
+  );
```

Comment:
This shouldn't need a `useMemo`!

Reply:
Hah! This was copy pasted from the other location it is used, will remove the memo, good catch!